internal/contributebot/README.md

```diff
@@ -13,3 +13,43 @@ incoming events are bursty.
 
 [github-async]: https://developer.github.com/v3/guides/best-practices-for-integrators/#favor-asynchronous-work-over-synchronous
 [github-ratelimit]: https://developer.github.com/v3/#rate-limiting
+
+## Setup
```

Comment:
Presumably nobody should actually do this, right? Maybe document where contributebot is running now?